lib/rest/accounts/v1/credential/aws.ts

```diff
@@ -130,10 +130,11 @@ export interface AwsContext {
    * @returns { Promise } Resolves to processed AwsInstance
```

Comment:
Is the `update` overload above here needed?

Reply:
Yup, I missed that out, it's not needed.